frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingSkjemaConst.tsx

```diff
@@ -140,5 +140,8 @@ export function defaultTiltaksgjennomforingData(
     opphav: tiltaksgjennomforing?.opphav ?? Opphav.MR_ADMIN_FLATE,
     fremmoteDato: fremmoteDatoFromTidspunkt(tiltaksgjennomforing?.fremmoteTidspunkt),
     fremmoteTid: fremmoteTidFromTidspunkt(tiltaksgjennomforing?.fremmoteTidspunkt),
+    deltidsprosent:
+      tiltaksgjennomforing?.deltidsprosent ??
+      (isTiltakMedFellesOppstart(avtale.tiltakstype.arenaKode) ? 100 : null),
```

Comment:
Har antatt at denne verdien kun skal settes for kurstiltak, men stemmer nok ikke. Fra Arena har alle gjennomføringer en verdi. Må nok beholde en default verdi, men at den er skjult for alle andre enn kurstiltak.